Exoft.Gamification.Api.Common/Models/Achievement/CreateAchievementModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Exoft.Gamification.Api.Common.Models.Achievement
+{
+    public class CreateAchievementModel
+    {
+        [Required]
```

Comment:
Please read task description correctly.... It was asked to skip validation for now.... Current validation is not valid